src/test/java/teammates/test/cases/logic/FeedbackQuestionsLogicTest.java

```diff
@@ -568,6 +595,23 @@ public void testGetFeedbackQuestionsForStudents() {
         assertEquals(actualQuestions, expectedQuestions);
     }
     
+    public void testGetFeedbackQuestionsForStudent() {
```

Comment:
`testGetFeedbackQuestionsForInstructor` should be updated too to test the custom feedback path